src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/TabularOverloadsBlanksAndErrors.txt

```diff
@@ -241,7 +241,7 @@ Error({Kind:ErrorKind.Numeric})
 Error({Kind:ErrorKind.Div0})
```

Comment:
>ind:ErrorKind.Div0 [](http://example.com/codeflow?start=8&length=18)

Why is this not {Kind:ErrorKind.Numeric}?

Reply:
The first error is on a scalar argument - so it is supposed to be distributed across the tabular arguments. The second one is a tabular argument, and since this is an error, this is what it is returned.

Reply:
This works differently in Power Apps though. Is it something we haven't fixed in PA yet?